tests/tests/tests.py

```diff
@@ -4,9 +4,13 @@
 """
 import itertools
 
+try:
+    from unittest import skipIf, skipUnless
+except ImportError:  # Python < 2.7
+    from django.utils.unittest import skipIf, skipUnless
```

Comment:
https://travis-ci.org/fusionbox/django-authtools/jobs/59913335#L238 here's the reason why. Let's revert it.